google/cloud/forseti/services/cli.py

```diff
@@ -1167,6 +1167,8 @@ def main(args,
                   'address.\n')
         else:
             print('Error occurred on the server side, message: {}'.format(e))
+    except Exception as e:  # pylint: disable=broad-except
```

Comment:
It would be awesome not to handle the broad exception here.  Instead, raise a custom exception, something like `ModelNotSetException`, from the `require_model()`, and then handle it here with a nicer message to tell the user to set the model first.

Reply:
Updated